comp/dogstatsd/server/parse.go

```diff
@@ -50,16 +50,17 @@ type parser struct {
 	readTimestamps bool
 }
 
-func newParser(cfg config.Reader, float64List *float64ListPool) *parser {
+func newParser(cfg config.Reader, float64List *float64ListPool, workerNum int) *parser {
 	stringInternerCacheSize := cfg.GetInt("dogstatsd_string_interner_size")
 	readTimestamps := cfg.GetBool("dogstatsd_no_aggregation_pipeline")
 
-	return &parser{
-		interner:         newStringInterner(stringInternerCacheSize),
+	p := &parser{
```

Comment:
Did you use a var for debugging (debugger / to print it out)? You did the same with the worker. Don't bother about my comment if it's just a style preference.